07-statistical-testing.Rmd

```diff
@@ -1 +1,570 @@
-# Statistical testing {#c07}
\ No newline at end of file
+# Statistical testing {#c07}
+
+When analyzing results from a survey, the point estimates described in Chapter \@ref(c06) are helpful for understanding the data at a high level, but researchers and the public often want to make comparisons between different groups. These comparisons are calculated through statistical testing where we compare the point estimates and the variance estimates of each statistic to see if there are statistically significant differences. The general idea of statistical testing is the same for data obtained through surveys and data obtained through other methods, but the importance lies in ensuring the variance is calculated correctly. Functions in the {survey} packages allow for the correct estimation of the variances. This chapter will cover the following statistical tests with survey data and functions:
+
+-  Comparison of proportions `svyttest()`
+-  Comparison of means `svyttest()`
+-  Goodness of fit tests `svygofchisq()`
+-  Tests of independence `svychisq()`
+-  Tests of homogeneity `svychisq()`
+
+Up to this point, the functions that we've provided have used the wrappers from the {srvyr} package. This means that the functions work with tidyverse syntax. However, the functions in this chapter do not have wrappers from the {srvyr} package and are instead pulled directly from the {survey} package. This means that the design object is *not* the first argument and that to use these functions with the magrittr pipe `%>%` and tidyverse syntax we will need to use dot (`.`) notation. Functions that work with the magrittr pipe `%>%` have the first argument as data. When data is run, it automatically places anything to the left of the pipe into the first argument of the function to the right of the pipe. For example, if we wanted to take the `mtcars` data and filter to only cars with 6 cylinders we can write the code in one of three ways:
+
+1. `filter(mtcars, cyl == 6)`
+2. `mtcars %>% filter(cyl == 6)`
+3. `mtcars %>% filter(., cyl == 6)`
+
+Each of these lines of code will produce the same output since the argument that takes the data is in the first spot in `filter()`. Those who have worked with the tidyverse, the first two are probably familiar. The third option functions the same way as the second one, but is explicit about where the data is going. Here we're telling R to take what's on the left side of the pipe `mtcars` and pipe it into the spot with the dot (`.`)---the first argument.
+
+In functions that are not part of the tidyverse, the data argument may be in a different spot in the functions. For example, in `svyttest()` the data argument is in the second spot, which means we need to place the dot (`.`) in the second spot and not the first. For example:
+
+```{r stattest-dotex}
+#| label: stattest-dotex
+#| eval: FALSE
+svydata_des %>% 
+  svyttest(x~y,.)
+```
```

Comment:
Change to code chunk, makes easier to run interactive with (run all above)

````suggestion
```
#| label: stattest-dotex

svydata_des %>% 
  svyttest(x~y,.)
```
````